banzai_floyds/fringe.py

```diff
@@ -97,13 +98,16 @@ class FringeCorrector(Stage):
     def do_stage(self, image):
         # Only divide the fringe out where the divisor is > 0.1 so we don't amplify
         # artifacts due to the edge of the slit
```

Comment:
The fringing stage takes a long time so I added some logs to tell the user it is still alive.